jsonHomeWork.playground/Contents.swift

```diff
@@ -0,0 +1,13 @@
+import Foundation
+
+//собираем url
+var components = URLComponents()
+components.scheme = "https"
+components.host = "api.magicthegathering.io"
+components.path = "/v1/cards"
+
+components.queryItems = [URLQueryItem(name: "name", value: "black_lotus|opt")]
+guard let requestURL = components.url else { fatalError("some error") }
+
+//запуск программы
+getData(urlRequest: requestURL.absoluteString)
```

Comment:
Ты собираешь ссылку в формате `url`, потом конвертируешь в строку, а потом в методе `getData` из строки обратно в `url`. Что-то не логично.
```suggestion

//запуск программы
getData(urlRequest: components.url)
```